pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -591,6 +592,7 @@ def __resolve_ckpt_dir(self, trainer: "pl.Trainer") -> None:
                 if isinstance(trainer.logger.version, str)
                 else f"version_{trainer.logger.version}"
             )
+            # TODO: Find out what ckpt_path should be with multiple loggers
```

Comment:
What do we want `ckpt_path` to be in the case of multiple loggers? Currently `LoggerCollection` runs a concatenation in `logger.name` and `logger.version`, but this might not be the best approach here.